src/ui/tournament/detail/tournamentView.tsx

```diff
@@ -274,23 +274,34 @@ function renderNavButton(icon: string, isEnabled: boolean, action: () => void) {
   })
 }
 
+function renderPlayerTitle(player: {title?: string}): Mithril.Child {
+  if (player.title == null) {
+    return null
+  }
+
+  return h('span.userTitle', [player.title, h.trust('&nbsp;')])
+}
+
 function renderPlayerEntry(userName: string, player: StandingPlayer, i: number, teamColor?: number, teamName?: string) {
   const evenOrOdd = i % 2 === 0 ? 'even' : 'odd'
   const isMe = player.name === userName
-  const ttc = teamColor ? teamColor : 0
+  const ttc = teamColor ?? 0
 
   return (
     <li key={player.name} data-player={player.name} className={`list_item tournament-list-item ${evenOrOdd}` + (isMe ? ' tournament-me' : '')} >
       <div className="tournamentIdentity">
-        <span className="flagRank" data-icon={player.withdraw ? 'b' : ''}> {player.withdraw ? '' : (player.rank + '.')} &thinsp; </span>
-        <span className="playerName"> {player.name + ' (' + player.rating + ')'}</span>
-        <span className={'playerTeam ttc-' + ttc}> {teamName ? teamName : '' } </span>
+        <span className="flagRank" data-icon={player.withdraw === true ? 'b' : ''}> {player.withdraw === true ? '' : (`${player.rank}.`)} &thinsp; </span>
+        <span className="playerName">
+          {renderPlayerTitle(player)}
+          {`${player.name} (${player.rating})`}
+        </span>
+        <span className={`playerTeam ttc-${ttc}`}> {teamName ?? ''} </span>
       </div>
       <div className={'tournamentPoints ' + (player.sheet.fire ? 'on-fire' : 'off-fire')} data-icon="Q">
         {player.score}
       </div>
     </li>
-  )
+  ) as Mithril.Child
```

Comment:
Why casting here? It should be not necessary. It's preferable to put the function return type explicitly then.

Reply:
Typescript doesn't seems to recognize JSX as Mithril.Vnode / Mithril.Child. The cast was to remove the "`any` return type" lint warning. A better long-term solution will be to tell Typescript about JSX or convert it all to `h` calls but that's a different PR :)

Reply:
ah ok thanks. I didn't notice this warning actually